models/lease.py

```diff
@@ -10,5 +12,5 @@ class LeaseModel(BaseModel):
     propertyID = db.Column(db.Integer, db.ForeignKey('properties.id'), nullable=False)
     tenantID = db.Column(db.Integer, db.ForeignKey('tenants.id'), nullable=False)
     occupants = db.Column(db.Integer)
-    dateTimeStart = db.Column(db.DateTime, nullable=False)
-    dateTimeEnd = db.Column(db.DateTime, nullable=False)
+    dateTimeStart = db.Column(db.DateTime, nullable=False, default=datetime.utcnow)
+    dateTimeEnd = db.Column(db.DateTime, nullable=False, default=datetime.utcnow)
```

Comment:
I believe I recently removed these defaults after talking to Hugh about them. What is the reason for adding them back in? I believe dateTimeStart and dateTimeEnd should be required fields when creating a lease, when there is a default assigned then we are essentially saying they're optional fields. If the requirement is for them to be optional then this is fine; however I can't imagine that be the case.

Reply:
Oh you're right, I think I misheard during the meeting. I'll fix it!